webapp/src/ts/services/contact-view-model-generator.service.ts

```diff
@@ -240,6 +241,10 @@ export class ContactViewModelGeneratorService {
   }
 
   loadChildren(model, options?) {
```

Comment:
Maybe: `loadChildrenOutsideZone` or leave the callback body inside the `ngZone.runOutsideAngular`


Same with: 
- _loadReports
- _getContact